pylutron/__init__.py

```diff
@@ -98,7 +99,15 @@ def _do_login_locked(self):
     self._telnet.write(self._user + b'\r\n')
     self._telnet.read_until(LutronConnection.PW_PROMPT)
     self._telnet.write(self._password + b'\r\n')
-    self._telnet.read_until(LutronConnection.PROMPT)
+    self._telnet.read_until(b'\n')
+    prompt = self._telnet.read_until(LutronConnection.PROMPT)
```

Comment:
why not read until prompt, but then strip the leading whitespace?

Reply:
```
-    self._telnet.read_until(b'\n')
-    prompt = self._telnet.read_until(LutronConnection.PROMPT)
+    prompt = self._telnet.read_until(LutronConnection.PROMPT).lstrip()
```